listenbrainz/webserver/templates/user/scraper.js

```diff
@@ -435,11 +442,16 @@ function updateLatestImportTimeOnLB() {
             final_msg += "<span><a href={{ url_for('user.profile', user_name = user_name) }}>Close and go to your ListenBrainz profile</a></span><br>";
             final_msg += "<span style='font-size:8pt'>Successfully submitted " + countReceived + " listens to ListenBrainz."
                 + " Please note that some of these listens might be duplicates leading to a lower listen count on LB.</span></br>";
-            if (notIncrementalImport == false && playCount != -1 && countReceived != playCount) {
+
+            // if the count received is different from the api count, show a message accordingly
+            // also don't show this message if it's an incremental import, because countReceived
+            // and playCount will be different by definition in incremental imports
+            if (!incrementalImport && playCount != -1 && countReceived != playCount) {
```

Comment:
The logic was wrong earlier here. We only need to show the inconsistency message if it is not an incremental import. That's why I changed the name of the variable, it confused me initially.